src/config/dynamic-theme-fixes.config

```diff
@@ -14248,6 +14248,18 @@ INVERT
 
 ================================
 
+wealthfront.com
+
+INVERT
+.dashboard-path-goal-icon-component
+
+CSS
+.default__9vSXGQvh8MlK1JxJBq-k4 {
+    background-color: rgba(51, 42, 173, 0.2) !important;
```

Comment:
`rgba(51, 42, 173, 0.2)`, could this be in the form of `${original color}`?

Reply:
I'm not sure what you mean by `${original color}`. I just found the right color against the background and with a different transparency.

Reply:
I meant if you replace `original color` with the color that the element has(e.g. `#fef` so it becomes `${#fef}`) dark reader will modify that color according to the user's settings. We try to avoid hardcoded colors as much as possible.